cfpb/cfgov-refresh
# Upgrade chart builder to v2.0.3 with cf-theme-overrides.less fix

This release fixes bugs reported in the following places: 

- GHE/CFGOV/platform/issues/1040
- messed up table header colors in https://github.com/cfpb/cfgov-refresh/pull/3231
<img width="704" alt="screen shot 2017-08-10 at 10 37 39 am" src="https://user-images.githubusercontent.com/1280430/29178695-f28d7a6e-7db7-11e7-9fcd-4f2a26c13de4.png">

Other changes included in this release are documented here: https://github.com/cfpb/cfpb-chart-builder/releases/tag/v2.0.3

## Testing

- check out branch, run `./setup.sh`
- `./runserver.sh`
- Verify that [this page](http://localhost:8000/about-us/the-bureau/) uses gray-5 (#f7f8f9) in the featured content module, and not gray-10 (#e7e8e9). see screenshots below for wrong and right gray examples.

## Screenshots
before
![screen shot 2017-08-09 at 1 25 51 pm](https://user-images.githubusercontent.com/702526/29192941-5890416e-7df1-11e7-8d2d-241d5a2ad7a6.png)


after
![screen shot 2017-08-09 at 1 26 23 pm](https://user-images.githubusercontent.com/702526/29192945-5d8dafd0-7df1-11e7-8183-bd8e037ba496.png)


## Checklist
